keps/sig-scheduling/1451-multi-scheduling-profiles/README.md

```diff
@@ -305,12 +293,168 @@ Note that we don't require a feature gate as users already have to opt-in by usi
+      
+      No
+
+* **Does enabling the feature change any default behavior?**
+
+  No, as long as a profile with the name of `default-scheduler` is kept.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  
+  Yes.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+  N/A.
+
+* **Are there any tests for feature enablement/disablement?**
+
+  There are tests (unit and integration) exercising single and multiple profiles.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+
+  The scheduler errors and exits during start up. Existing workloads are not
+  affected.
+
+* **What specific metrics should inform a rollback?**
+
+  Metric "schedule_attempts_total" remaining at zero when new pods are added.
+
+* **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
+
+  N/A.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features,
+  APIs, fields of API types, flags, etc.?**
+
+  No.
+
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+
+  - Scheduling metrics will have a new field for the profile name. These are:
+    - "schedule_attempts_total"
+    - "e2e_scheduling_duration_seconds"
+    - "scheduling_algorithm_duration_seconds"
+    - "scheduling_algorithm_preemption_evaluation_seconds"
+    - "binding_duration_seconds"
+  - Pods have scheduling Events with different scheduler names.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to
+  determine the health of the service?**
+  
+  - [x] Metrics
+    - Metric name: `schedule_attempts_total`, `e2e_scheduling_duration_seconds`
+    - Components exposing the metric: `kube-scheduler`
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+
+  No new SLO.
+  However, any scheduler related SLOs that can be defined in other KEPs or
+  individually by cluster operators, can be split by profile, using the metrics
+  discussed above.
+
+* **Are there any missing metrics that would be useful to have to improve
+  observability if this feature?**
+  
+  No.
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+
+  No.
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+
+  No.
+
+* **Will enabling / using this feature result in introducing new API types?**
+
+  No.
+
+* **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+  
+  No.
+
+* **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+  
+  No.
+
+* **Will enabling / using this feature result in increasing time taken by any
+  operations covered by [existing SLIs/SLOs][]?**
+  
+  No.
+
+* **Will enabling / using this feature result in non-negligible increase of
```

Comment:
We can also mention that users who don't want to use this feature won't get penalties - single profile basically won't increase resource usage.

Reply:
Done